### DISCUSSION TITLE
process Placeholder

### DISCUSSION THREAD

**Discussion by ansDArnold** [ansys/pyoptislang#63] [Q&A]
Hi PyOptiSLang,

just thinking if there is already a way to process an optiSLang placeholder definition file for running an optiSLang job thru PyOptiSLang.

Does anyone have an idea how this could be done?

Thanks,
Daniel

**Comment by rfahlberg** [comment]
@ansDArnold Currently implementing possibility to pass additional/generic oSL command line arguments to "Optislang" and "OptislangServerProcess" class constructors. This should probably be sufficient as long as we don't have the explicit interface.

**Reply by rfahlberg** [reply]
@ansDArnold Covered in https://github.com/ansys/pyoptislang/pull/137
